test/hexpm/web/controllers/login_controller_test.exs

```diff
@@ -1,6 +1,5 @@
 defmodule Hexpm.Web.LoginControllerTest do
-  use Hexpm.ConnCase, async: true
-  alias Hexpm.Accounts.Users
+  use Hexpm.ConnCase
```

Comment:
async: true seems to work for me, have you run into any issues like the ones with bamboo?